src/pods/edit-realtion/edit-relation.business.ts

```diff
@@ -0,0 +1,16 @@
+import { DropdownOptionVm } from '@/common/components';
+import { DatabaseSchemaVm } from '@/core/providers/canvas-schema';
+
+export const mapRelationsTipeToDropdonwVm = (): DropdownOptionVm[] => [
+  { id: '1', label: '1:1' },
+  { id: '2', label: '1:M' },
+  { id: '3', label: 'M:1' },
+];
+
+//Test, it's possible that it arrives empty?
+export const mapTablesToDropdonwVm = (
```

Comment:
Yes when you start a new empty diagram, I'm going to add a new issue for this:
- When there are no table this mapper shoud behave and add unit tests.
- Relations toolbar button should be disable.

https://github.com/Lemoncode/mongo-modeler/issues/91

Reply:
Added comment for the TODO task